src/mixed-reality/azext_mixed_reality/tests/latest/test_spatial_anchors_account.py

```diff
@@ -0,0 +1,75 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure.cli.testsdk import (ScenarioTest, ResourceGroupPreparer)
+from knack.util import CLIError
+
+
+class SpatialAnchorsAccountScenarioTest(ScenarioTest):
+
+    Location = 'eastus2'
+
+    @ResourceGroupPreparer(location=Location)
+    def test_spatial_anchors_account_scenario(self, resource_group):
+
+        initial = 'az spatial-anchors-account'
+
+        def assert_spatial_anchors_account_as_expected(cmd):
+
+            result = self.cmd(cmd).get_output_in_json()
+            self.assertTrue(name, result['name'])
```

Comment:
Instead of using self.assertTrue, you can use checks=self.Check() or checks=[self.Check(), self.Check()] in conjunction with self.cmd().

See [here](https://github.com/Azure/azure-cli/blob/dev/doc/authoring_tests.md#sample-3-validate-the-return-json-value-using-jmespath) for more info.

Try to assert any other properties that make sense to you here and in your keys test.